spec/lib/payment_calculator/turnip_features/ecf/uplift_payment.feature

```diff
@@ -0,0 +1,8 @@
+@ecf
+Feature: ECF contract calculations
+
+  Scenario: Uplift payment
+    Given An uplift per participant of £100
+    And there are 300 participants who have started that are eligible for the uplift payment
```

Comment:
maybe we should split this line into 100 sparsity and 200 pupil premium

Reply:
Added in the dfdc589